Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2570,6 +2568,195 @@ def pin_message():
+                'FullName': full_name,
+                'TimeStamp': message.get('ts'),
+                'HasReplies': has_replies,
+                'ThreadTimeStamp': thread_ts
+            }
+            context.append(entry)
+        readable_output = tableToMarkdown(f'Channel details from Channel ID - {channel_id}', context)
+    else:
+        raise DemistoException(f'An error occurred while listing conversation history: {raw_response.get("error")}')
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': messages,
+        'EntryContext': {'Slack.Messages': context},
+        'ContentsFormat': formats['json'],
+        'HumanReadable': readable_output,
+        'ReadableContentsFormat': formats['markdown']
+    })
+
+
+def conversation_replies():
+    """
+    Retrieves replies to specific messages, regardless of whether it's
+    from a public or private channel, direct message, or otherwise.
+    """
+    args = demisto.args()
+    channel_id = args.get('channel_id')
+    context: list = []
+    readable_output: str = ''
+    body = {
+        'channel': channel_id,
+        'ts': args.get('thread_timestamp'),
+        'limit': arg_to_number(args.get('limit'))
+    }
+    raw_response = send_slack_request_sync(CLIENT, 'conversations.replies', http_verb="GET", body=body)
+    messages = raw_response.get('messages', '')
+    if not raw_response.get('ok'):
+        error = raw_response.get('error')
+        return_error(f'An error occurred while listing conversation replies: {error}')
+    if isinstance(messages, dict):
+        messages = [messages]
+    if isinstance(messages, list):
+        for message in messages:
+            reply_count = 'No'
+            name = 'N/A'
+            full_name = 'N/A'
+            if 'subtype' not in message:
+                user_id = message.get('user')
+                body = {
+                    'user': user_id
+                }
+                user_details_response = send_slack_request_sync(CLIENT, 'users.info', http_verb="GET", body=body)
+                user_details = user_details_response.get('user')
+                name = user_details.get('name')
+                full_name = user_details.get('real_name')
+            if 'reply_count' in message:
+                reply_count = 'Yes'
+            entry = {
+                'Type': message.get('type'),
+                'Text': message.get('text'),
+                'UserId': message.get('user'),
+                'Name': name,
+                'FullName': full_name,
+                'TimeStamp': message.get('ts'),
+                'ThreadTimeStamp': message.get('thread_ts'),
+                'IsParent': reply_count
+            }
+            context.append(entry)
+        readable_output = tableToMarkdown(f'Channel details from Channel ID - {channel_id}', context)
+    else:
+        error = raw_response.get('error')
+        return_error(f'An error occurred while listing conversation replies: {error}')
```

Comment:
why sometimes `return_error` and sometimes `DemistoException`? We should only raise

Reply:
agree